mindmap/public/js/src/mindmap.js

```diff
@@ -84,7 +85,8 @@ function MindMapXBlock(runtime, element, context) {
                 gettext("Username"),
                 gettext("Uploaded"),
                 gettext("Submission Status"),
-                gettext("Grade"),
+                gettext("Raw score"),
+                gettext("Weighted score"),
```

Comment:
I think these two columns needs translations and update the translations lines

Reply:
Done!